File: gabriel/Problema0322/Problema0322.java
package Problema0322;

import java.util.Scanner;
public class Problema0322 {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        
         System.out.println("Digite uma mensagem: ");
         String mensagem = input.nextLine();
         
         String mensagemLower = mensagem.toLowerCase();
         
         int contA=0, contE=0, contI=0, contO=0, contU=0;
         char vetorMensagem[] = mensagemLower.toCharArray();
         
         for(int i=0; i<mensagem.length(); i++){
             switch(vetorMensagem[i]){
                 
                 case 'a':
                     contA = contA + 1;
                     break;
                     
                 case 'e': 
                     contE = contE + 1;
                     break;
                     
                 case 'i': 
                     contI = contI + 1;
                     break;
                     
                 case 'o':
                     contO = contO + 1;
                     break;
                     
                 case 'u': 
                     contU = contU + 1;
                     break;
             }
         }
         
         System.out.println("Quantidade de 'a' e 'A' na mensagem digitada: " + contA);
         System.out.println("Quantidade de 'e' e 'E' na mensagem digitada: " + contE);
         System.out.println("Quantidade de 'i' e 'I' na mensagem digitada: " + contI);
         System.out.println("Quantidade de 'o' e 'O' na mensagem digitada: " + contO);
         System.out.println("Quantidade de 'u' e 'U' na mensagem digitada: " + contU);
     }
     
 }
 

File: gabriel/Problema0322/Problema0322.py
mensagem = input("Digite uma mensagem: ")
mensagemLower = mensagem.lower()

contA = 0; contE = 0; contI = 0; contO = 0; contU = 0;

for i in range(len(mensagem)):
  if(mensagemLower[i] == "a"):
    contA = contA + 1

  elif(mensagemLower[i] == "e"):
    contE = contE + 1

  elif(mensagemLower[i] == "i"):
    contI = contI + 1

  elif(mensagemLower[i] == "o"):
    contO = contO + 1

  elif(mensagemLower[i] == "u"):
    contU = contU + 1

print("Quantidade de 'a' e 'A' na mensagem digitada: " + str(contA))
print("Quantidade de 'e' e 'E' na mensagem digitada: " + str(contE))
print("Quantidade de 'i' e 'I' na mensagem digitada: " + str(contI))
print("Quantidade de 'o' e 'O' na mensagem digitada: " + str(contO))
print("Quantidade de 'u' e 'U' na mensagem digitada: " + str(contU))